src/main/java/com/theelixrlabs/healthcare/constants/PatientConstants.java

```diff
@@ -7,15 +7,17 @@ public class PatientConstants {
     public static final String AADHAAR_FORMAT_PATTERN = "%s %s %s";
     public static final String AADHAAR_NUMBER_EXISTS_KEY = "patientAadhaarNumber.exists";
     public static final String AADHAAR_NUMBER_REGEX = "^[2-9][0-9]{11}";
-    public static final String ALPHA_CHARACTERS_ONLY_REGEX = "^[a-zA-Z]+$";
+    public static final String ALPHA_CHARACTERS_REGEX = "^[a-zA-Z]+$";
     public static final String CREATE_PATIENT_ENDPOINT = "/addPatient";
-    public static final String FIRST_NAME_NOT_EMPTY_KEY = "firstName.notEmpty";
-    public static final String GET_PATIENT_BY_ID_ENDPOINT = "/getPatientById/{id}";
+    public static final String FIRST_NAME_NOT_EMPTY_KEY = "firstName.not.empty";
     public static final String INVALID_FIRST_NAME_KEY = "firstName.invalid";
     public static final String INVALID_LAST_NAME_KEY = "lastName.invalid";
     public static final String INVALID_UUID_KEY = "uuid.invalid";
-    public static final String LAST_NAME_SHOULD_NOT_EMPTY_KEY = "lastName.notEmpty";
-    public static final String MESSAGE_RESOURCE_CLASSPATH_NAME="classpath:messages";
+    public static final String LAST_NAME_SHOULD_NOT_BE_EMPTY_KEY = "lastName.not.empty";
```

Comment:
Make sure properties file key name is also matching to the message and variable name
lastName.should.not.be.empty

Reply:
resolved.